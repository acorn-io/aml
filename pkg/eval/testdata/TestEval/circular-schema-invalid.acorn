schema {
    top: {
        x: 1
        y: top
    }
}


top: x: 1
