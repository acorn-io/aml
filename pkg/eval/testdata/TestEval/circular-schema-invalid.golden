`schema violation .top: key not found "top": exceeded max scope depth 101 > 100: circular-schema-invalid.acorn:4:12 (circular-schema-invalid.acorn:2:10) (circular-schema-invalid.acorn:2:10) (circular-s (circular-schema-invalid.acorn:1:8)`
